content/blog/2024/02/camel44-whatsnew/index.md

```diff
@@ -53,10 +53,11 @@ And you can find a small example here using variables with EIPs:
 
 The JBang debugger can also show variables if enabled with `--show-exchange-variables`, as shown in the screenshot below:
 
-[source,bash]
-----
+
+```bash
 camel debug * --show-exchange-variables
-----
```

Comment:
@davsclaus do you mind double checking this, please? This was rendered incorrectly on the blog post.